### PR TITLE
Make sure that all the accepted_flowers listed in bees.json correspond to entries of flowers.json

### PR DESCRIPTION
**Pull request by helloworld12321**
Previously, one flower in bees.json, `taracaxum_officinale`, was not listed in flowers.json. (This is just a typo—`taraxacum_officinale` is listed in flowers.json. :upside_down_face:) This PR fixes the problem.

Also, to stave off future typos, this PR adds tests to make sure that the accepted_flowers in bees.json exist in flowers.json.

### CODE REVIEW

File: .github/workflows/validate-json.yml [hunk 1]
```diff
@@ -7,32 +7,40 @@ on:
...
+          INPUT_JSONS: ./nests.json
 
-  validate-nests:
+  # Adapted from
+  # https://docs.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions
+  run-tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
```
Comment (floogulinc) [inline .github/workflows/validate-json.yml:36]: If we are just using one version we don't need this.

File: .github/workflows/validate-json.yml [hunk 2]
```diff
@@ -7,32 +7,40 @@ on:
...
-      - name: Validate JSON
-        uses: docker://orrosenblatt/validate-json-action:latest
-        env: 
-          INPUT_SCHEMA: ./nests.schema.json
-          INPUT_JSONS: ./nests.json  
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```
Comment (floogulinc) [inline .github/workflows/validate-json.yml:42]:
```suggestion
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 12.x
```

**Comment by floogulinc** [comment]
We should probably add the specs directory to the list of places it looks for changes to trigger the tests:
```yaml
on:
  push:
    paths:
    - '**.json'
    - '.github/workflows/validate-json.yml'
    - 'spec/**'
```

**Comment by helloworld12321** [comment]
Good catch, thanks!

**Pull request merged**